tools/gcs/upload.sh

```diff
@@ -40,32 +60,31 @@ fi
 #   https://storage.googleapis.com/envoy-pr/28462/docs/index.html
 #
 
-UPLOAD_PATH="$(git rev-parse HEAD | head -c7)"
-REDIRECT_PATH="${SYSTEM_PULLREQUEST_PULLREQUESTNUMBER:-${BUILD_SOURCEBRANCHNAME}}"
+UPLOAD_PATH="$(git -C "${ENVOY_SOURCE_DIR}" rev-parse HEAD | head -c7)"
 GCS_LOCATION="${GCS_ARTIFACT_BUCKET}/${UPLOAD_PATH}/${TARGET_SUFFIX}"
 
 echo "Uploading to gs://${GCS_LOCATION} ..."
-bazel "${BAZEL_STARTUP_OPTIONS[@]}" run "${BAZEL_BUILD_OPTIONS[@]}" \
-      //tools/gsutil \
-      -- -mq rsync \
-         -dr "${SOURCE_DIRECTORY}" \
+"${GSUTIL}" \
+         -mq rsync \
+         -dr "${UPLOAD_DIRECTORY}" \
          "gs://${GCS_LOCATION}"
 
+if [[ -z "${REDIRECT_PATH}" ]]; then
+    echo "Artifacts uploaded to: https://storage.googleapis.com/${GCS_LOCATION}" >&2
```

Comment:
doesn't this need `/index.html` after the GCS_location (similar to the link below)?
I'm guessing that the default will just redirect/show the index.html page, but just want to make sure.

Reply:
no - in this case the index.html doesnt exist (or at least most likely doesnt), so would not work

that said, the links here dont work 100% as expected - in this PR i just wanted to move the code and keep existing behaviour - the reason being, that most of the usage can be just removed in the next PR i want to land, and i figured i can clean it up after that